hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -200,6 +209,10 @@ public NearCacheStatsImpl getNearCacheStats() {
         return stats;
     }
 
+    @Override
+    public void preload(DataStructureAdapter adapter) {
```

Comment:
This implementation is going to be removed, just had to add the method until this PR is merged: https://github.com/hazelcast/hazelcast/pull/8997